From 5a019410a0908d7a66a0526a47bed1883c02c5fe Mon Sep 17 00:00:00 2001
From: Abel Vesa <abel.vesa@nxp.com>
Date: Thu, 9 Jan 2020 15:20:27 +0200
Subject: [PATCH] LF-794-4 gpu: cdn: mhdp-common: Increase the check alive
 timeout

It seems that the firmware on i.MX8QM starts a little bit
after 20us after the ucpu has been relased (un-resetted).
Therefore, increase the max timeout for check alive to 50us.

Signed-off-by: Abel Vesa <abel.vesa@nxp.com>
Reviewed-by: Sandor Yu <sandor.yu@nxp.com>
Acked-by: Wen He <wen.he_1@nxp.com>
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
---
 drivers/gpu/drm/bridge/cadence/cdns-mhdp-common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/bridge/cadence/cdns-mhdp-common.c
+++ b/drivers/gpu/drm/bridge/cadence/cdns-mhdp-common.c
@@ -1039,7 +1039,7 @@ EXPORT_SYMBOL(cdns_mhdp_read_hpd);
 bool cdns_mhdp_check_alive(struct cdns_mhdp_device *mhdp)
 {
 	u32  alive, newalive;
-	u8 retries_left = 10;
+	u8 retries_left = 50;
 
 	alive = cdns_mhdp_bus_read(mhdp, KEEP_ALIVE);
 
